src/components/Card/CardVideo.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import { CardImageProps } from '../Card'
+import { CardImageProps } from './Card'
 
 export interface CardVideoProps {
```

Comment:
Please use the Icon component for the video icon. If its not there, added to the icon library.

Reply:
Fixed, added Icon component